webapp/channels/src/components/channel_header_dropdown/menu_items/close_message/close_message.tsx

```diff
@@ -107,6 +107,7 @@ export default class CloseMessage extends React.PureComponent<Props> {
                 show={channel.type === Constants.DM_CHANNEL || channel.type === Constants.GM_CHANNEL}
                 onClick={this.handleClose}
                 text={text}
+                icon={<CloseIcon color='#808080'/>}
```

Comment:
Does this color work through the different themes?

Reply:
The color works on all themes.Do you need any other functionality?

Reply:
Yeah the icons are visible and clear.As you said, I am looking forward to further review.